build/verify-archive.sh

```diff
@@ -9,7 +9,7 @@
 set -euo pipefail
 
 apt-get update
-apt-get install -y autoconf bison cmake libncurses-dev
+apt-get install -y autoconf bison cmake libncurses-dev psmisc
```

Comment:
I assume that's for `killall`? Maybe we could do away by using `pgrep cockroach | kill -TERM` instead?